libraries/Bluefruit52Lib/src/services/BLEHidGeneric.h

```diff
@@ -60,7 +60,6 @@ class BLEHidGeneric : public BLEService
 
     void enableKeyboard(bool enable);
     void enableMouse(bool enable);
-    void enableGamepad(bool enable);
```

Comment:
we should also need to keep this function, user may only want to do a single keyboard, or mouse, or just keyboard + mouse without gamepad